bot/task.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+
+class Complexity(Enum):
+  Easy = 10
+  Middle = 20
+  Hard = 30
+
+complexityDict = {
+  Complexity.Easy: 'Легкий',
+  Complexity.Middle: 'Средний',
+  Complexity.Hard: 'Тяжелый'
+}
+
+class Task(object):
+    def __init__(self, number, name, announcementLink, solutionlink=None, level=None, tags=None):
+        self.number = number
+        self.name = name
+        self.announcementLink = announcementLink
+        self.solutionlink = solutionlink
```

Comment:
Привести в соответствие: либо все через undercore, либо camelCase. Но не в одно слово.

```suggestion
        self.solutionLink = solutionLink
```

Reply:
переделал на underscore. CamelCase на pep-8 не рекомендуется на сколько я знаю